xcode/Subconscious/Shared/Components/FirstRun/RecoveryPhraseView.swift

```diff
@@ -43,11 +43,11 @@ struct RecoveryPhraseView: View {
 }
 
 enum RecoveryPhraseAction: Hashable {
-    case setPhrase(_ phrase: String)
+    case setPhrase(_ phrase: RecoveryPhrase?)
```

Comment:
Use RecoveryPhrase in action to prevent accidentally leaking string via action logs.